Style.css

```diff
@@ -325,3 +345,308 @@ button:disabled {
+  }
+
+  .page-flow > li > a {
+    text-decoration: none;
+  }
+
+  .page-flow > li > a:hover {
+    color: blue;
+  }
+
+  .headline {
+    margin-top: 5.5%;
+    background: url(./desktop-images/Header\ bg.png);
+    background-size: cover;
+    background-repeat: no-repeat;
+    padding: 70px 200px;
+    background-color: white;
+    height: 100vh;
+  }
+
+  h1 {
+    font-size: 48px;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+  }
+
+  .body-text {
+    font-size: 20px;
+    line-height: 28px;
+  }
+
+  .works {
+    padding: 0 5%;
+  }
+
+  .work-item {
+    width: 100%;
+    flex-direction: row;
+    gap: 30px;
+  }
+
+  .work-item:nth-child(even) {
+    flex-direction: row-reverse;
+  }
+
+  .primary-text {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .title {
+    font-size: 40px;
+    line-height: 52px;
+    color: #091e42;
+    margin-bottom: 0;
+  }
+
+  .title2,
+  .title3 {
+    visibility: hidden;
+    margin-bottom: -4vh;
+  }
+
+  .title2::after,
+  .title3::after {
+    content: "Facebook 360";
+    visibility: visible;
+    display: block;
+    position: relative;
+    top: -2vh;
+    left: 0;
+  }
+
+  .title3::after {
+    content: "Uber Navigation";
+  }
+
+  .description {
+    font-size: 16px;
+    line-height: 24px;
+    padding-right: 60px;
+  }
+
+  .frame-2 {
+    gap: 12px;
+    align-items: center;
+  }
+
+  .frame-end {
+    margin-left: -6%;
+  }
+
+  .ruby {
+    content: url("./desktop-images/Tag.png");
+  }
+
+  .client,
+  .details {
+    font-size: 18px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .details {
+    font-weight: 400;
+    color: #6b778c;
+  }
+
+  .client2,
+  .client4 {
+    visibility: hidden;
+  }
+
+  .client2::after,
+  .client4::after {
+    visibility: visible;
+    display: block;
+    position: relative;
+    top: -2vh;
+    left: 0;
+    content: "FACEBOOK";
+  }
+
+  .client4::after {
+    content: "UBER";
+    padding-inline-start: 30%;
+  }
+
+  .role,
+  .end-role {
+    visibility: hidden;
+  }
+
+  .role::after,
+  .end-role::after {
+    visibility: visible;
+    display: block;
+    position: relative;
+    top: -2vh;
+    left: 0;
+    content: "Full Stack Dev";
+  }
+
+  .end-role::after {
+    content: "Lead Developer";
+  }
+
+  .normal-button {
+    width: 130px;
+  }
+
+  .about {
+    background: url(./desktop-images/about\ myself\ -\ background-\ shape-\ desktop@2x.png);
+    flex-direction: row;
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: right;
+    gap: 80px;
+    padding: 0 5%;
+    margin-top: 15%;
+    padding-bottom: 10%;
+  }
+
+  .personal-info {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+  }
+
+  .personal-info > h1 {
+    font-size: 40px;
+    line-height: 52px;
+    color: #091e42;
+  }
+
+  .skill-list {
+    display: flex;
+    flex-direction: column;
+    padding: 0 70px;
+    justify-content: center;
+    align-items: center;
+  }
```

Comment:
- [ ] Please 🙏  I noticed also that you will perhaps need to position the carets (right and down arrow) properly according to the template for the mobile version. 👍